src/components/dialogs/parameters/sensi/sensitivity-analysis-parameters.jsx

```diff
@@ -101,7 +102,7 @@ export const SensitivityAnalysisParameters = ({
     const [isSubmitAction, setIsSubmitAction] = useState(false);
     const [analysisComputeComplexity, setAnalysisComputeComplexity] =
         useState(0);
-    const [providers, , , , params, ,] = parametersBackend;
+    const [providers, , , , params] = parametersBackend;
```

Comment:
???

Reply:
Lol prettier fixed this for me, what a beauty